docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1179,15 +1279,34 @@ handle field-level and form-level messages. This has the following properties:
 * `Requirements::delete_combined_files()` and `Requirements::delete_combined_files()` methods have been removed
  as they are obsolete.
 
-### <a name="overview-mailer"></a>Email and Mailer
+### <a name="overview-i18n"></a>i18n API
 
-* `Mailer` converted to an interface
-* `SwfitMailer` added as new default mailer
-* `Email` re-written to be powered by [SwiftMailer](https://github.com/swiftmailer/swiftmailer)
+#### <a name="overview-i18n-api"></a>i18n API Additions / Changes
+
+* Upgrade of i18n to symfony/translation
+* Localisation based on language-only (without any specific locale) is now supported
+* i18nEntityProvider::provideI18nEntities() Now is expected to return only a single array
+  map of key to default values.
+* i18n keys for '.PLURAL_NAME' and '.SINGULAR_NAME' have been changed back to FQN class names
+  for all DataObject subclasses.
+* i18n key for locale-respective pluralisation rules added as '.PLURALS'. These can be configured
+  within yaml in array format as per [ruby i18n pluralization rules](http://guides.rubyonrails.org/i18n.html#pluralization).
+
+#### <a name="overview-i18n-removed"></a>i18n API Removed API
 
+* Zend_Translate removed
+* i18n::_t $context parameter deprecated
+* i18n::_t Support for sprintf-style `%s` arguments deprecated
+* i18n::_t Using non-associative injection with named parameters is now an error
```

Comment:
Minor: Can you either wrap things like this in backticks or escape the markdown characters to prevent the formatting breaking?